Doc/library/enum.rst

```diff
@@ -808,6 +834,20 @@ Utilities and Decorators
 
    .. versionadded:: 3.11
 
+.. decorator:: global_enum
+
+   A decorator to change the *str()* and *repr()* of an enum to show the module
+   of the enum instead of its class.  Should only be used when the enum members
+   are exported to the global namespace (see *re.RegexFlag* for an example).
```

Comment:
```suggestion
   A decorator to change the :class:`str() <str>` and :func:`repr` of an enum
   to show its members as belonging to the module instead of its class.
   Should only be used when the enum members are exported
   to the module global namespace (see class:`re.RegexFlag` for an example).
```

* Tweaks to wording to clarify as previous
* Sphinxify